gunicorn/config.py

```diff
@@ -1309,6 +1309,19 @@ class LogConfig(Setting):
     """
 
 
+class LogConfigDict(Setting):
+    name = "logconfig_dict"
+    section = "Logging"
+    validator = validate_dict
+    default = {}
+    desc = """\
+    The log config dictionary to use, using the standard Python logging
+    module's dictConfig format added in python 2.7.
```

Comment:
dictConfig -> \`\`dictConfig\`\`

Reply:
We also need add a link to https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig